package.json

```diff
@@ -7,7 +7,7 @@
     "build": "babel src  --out-dir production",
     "start": "node production/index.js",
     "dev": "nodemon --exec babel-node src/index.js",
-    "coverage": "nyc report --reporter=text-lcov | coveralls",
+    "coverage": "nyc report --reporter=html",
```

Comment:
How can I verify that the coverage is being sent?

Reply:
By clicking on the coveralls badge you will be redirected to the coveralls web. All the passing branches are being automatically passed there. below is an example from GAD's branch
<img width="1173" alt="Screen Shot 2019-12-10 at 10 52 52" src="https://user-images.githubusercontent.com/52409462/70513960-47cf2680-1b3b-11ea-9018-a69f65ad51be.png">

Reply:
I'm more particular about this branch.